tilecloud_chain/wmts_get_capabilities.jinja

```diff
@@ -5,9 +5,14 @@
     xmlns:xlink="http://www.w3.org/1999/xlink"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xmlns:gml="http://www.opengis.net/gml"
-    xsi:schemaLocation="http://schemas.opengis.net/wmts/1.0/wmtsGetCapabilities_response.xsd">
-  <ows:ServiceIdentification> </ows:ServiceIdentification>
-  <ows:ServiceProvider> </ows:ServiceProvider>
+    xsi:schemaLocation="http://schemas.opengis.net/wmts/1.0/wmtsGetCapabilities_response.xsd">{%
+  if title %}
+  <ows:ServiceIdentification>
+    <ows:Title>{{title}}</ows:Title>
+    <ows:ServiceType>OGC WMTS</ows:ServiceType>
```

Comment:
This is so badly describe than I think that's should be configurable...

Reply:
That's possible indeed but is it really needed? All the examples I have found use this value.

Reply:
It can be the default value...